recipe/conda_build_config.yaml

```
@@ -0,0 +1,4 @@
+r_base:
+ - 4.2
+ - 4.1
+ - 4.0
```

Conda Forge does not support R 4.2 for **win** platform. Please follow two space indentation for YAML files.

You must rerender after merging this suggestion:

```suggestion
  - 4.2  # [not win]
  - 4.1
  - 4.0
```